### PR TITLE
Update global surface CX namelist

### LINKED ISSUE #155: Use explicit CX namelist for global surface observations
[opsinputs/pull/152](https://github.com/MetOffice/opsinputs/pull/152) introduces the CX namelist for global surface observations. @ctgh suggested that it would be clearer to explicitly list the CX fields requested rather than using the add/remove option.

This should be a straightforward change to address with no change to the result.

### PR DESCRIPTION
**Pull request by ctgh**
Explicitly list the fields in the global surface CX namelist. This follows the UKV namelist but adds three more required fields: cloud_bulk, cf and cl.

The existing opsinputs ctest is unchanged.

Sith global KGO output: http://fcm1/cylc-review/taskjobs/chthomas/?suite=sith-cx-global%2Frun3

(Not shown) sith UKV and malak global KGOs are also unaffected.

Fixes #155

### CODE REVIEW

**Review by JoanneAWaller — APPROVED** [review]
Thanks @ctgh for making the cx list explicit, it is much clearer than before.

File: etc/global/cx/Surface.nl [hunk 1]
```diff
@@ -1,5 +1,49 @@
 &CXControlNL
-cxfields_remove=23, 25, 31, 90, 1235, 3208, 3247, 3255, 3464, 3465, 8223, 8225, 9203, 9204, 9217, 9219, 16222, 20000, 20001, 20002
-cxfields_add=266, 267, 268
-
+! All variables below are found in OPS cx outputs for surface processing. Not all are
+! necessary for JOPA.
```
Comment (ReubenHill) [inline etc/global/cx/Surface.nl:3]: Has this actually been checked in this case? If not I think the comment should be amended. The list is useful, I just don't want us to imply we've confirmed these are the fields that are output by OPS when we haven't actually and cause confusion down the line!

**Review by ReubenHill — COMMENTED** [review]
Excellent! Just one comment to address

File: etc/global/cx/Surface.nl (hunk 1 above)
Comment (ctgh) [reply etc/global/cx/Surface.nl:3]: Good point - taking a look...

Comment (ctgh) [reply etc/global/cx/Surface.nl:3]: I have updated the table in be4a150. There were a few changes to make in the surface variables, and nothing for the upper-air ones.

**Review by ReubenHill — APPROVED** [review]
Thanks!

**Comment by ctgh** [comment]
Thanks for both of your reviews!

**Pull request merged**